src/static/json/PhysicsRoutines/BetaFromESingle.json

```diff
@@ -1,15 +1,14 @@
 {
   "visibleName": "Relativistic β from energy",
   "functionName": "AT_beta_from_E_single",
-  "description": "Relativistic speed β (v/c) of particle as a function of energy per nucleon",
+  "description": "Relativistic speed β \\(\\frac{v}{c}\\) of particle as a function of energy per nucleon",
   "plot": false,
   "resultPrecision": 4,
   "formItems": [
     {
       "type": "input",
       "parameterName": "E_MeV_u",
-      "label": "Energy [MeV/amu]",
-      "placeholder": "10",
+      "label": "Energy \\(\\frac{MeV}{amu}\\)",
```

Comment:
Does it support math mode without italics ?
Sth like `$\frac{\textrm{MeV}}{\textrm{amu}}$` in LaTeX

Reply:
\\(\\frac{\\mathrm{MeV}}{\\mathrm{amu}}\\)
https://libamtrack.github.io/web_dev/#Calculators/%CE%B2fromenergy